### PR TITLE
Feature/release notes 9.21.0

### PR DESCRIPTION
**Pull request by marijnritense**

### CODE REVIEW

File: release-notes/major9/09.21.0/valtimo-backend-libraries.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+The following was deprecated:
+
+* **FormFieldDataResolver `get` method**
+
+  The method `get(String documentDefinitionName, UUID documentId, String... varNames)` was deprecated and \
+  is replaced with `get(DataResolvingContext dataResolvingContext, String... varNames)`.
+
+* **Deprecation2**
+
+  X was deprecated and is replaced with Y.
```
Comment (mritense) [inline release-notes/major9/09.21.0/valtimo-backend-libraries.md:56]: This file should be cleaned up

File: release-notes/major9/09.21.0/valtimo-frontend-libraries.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+  The `v-button in @valtimo/user-interface` was not able to handle async values since it was not listening to any changes.
+
+## Breaking changes
+
+The following breaking changes were introduced:
+
+* **Breaking change1**
+
+* **Breaking change2**
```
Comment (mritense) [inline release-notes/major9/09.21.0/valtimo-frontend-libraries.md:85]: This file should be cleaned up

**Review by mritense — COMMENTED** [review]
Merge conflict with main branch?

File: release-notes/major9/09.21.0/valtimo-backend-libraries.md (hunk 1 above)
Comment (marijnritense) [reply release-notes/major9/09.21.0/valtimo-backend-libraries.md:56]: ...I'm an idiot, will fix it

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**